index.html

```diff
@@ -9,5 +9,14 @@
   <body>
     <h1>Hello, world!</h1>
     <p class="text">Olá, mundo</p>
+    <section>
+      <h2>Coisas que gosto de fazer!</h2>
+      <ul>
+        <li class="text">Trabalhar</li>
+        <li class="text">Academia</li>
+        <li class="text">Comer</li>
+        <li class="text">Férias</li>
```

Comment:
```suggestion
        <li class="text">Trabalhar</li>
        <li class="text">Academia</li>
        <li class="text">Comer</li>
        <li class="text">Férias</li>
```
Essa classe está sendo utilizada no paragrafo, tem certeza que deveriam utilizar as mesmas classes?

Reply:
Os estilos são os mesmos, entçao as classes podem ser a mesma.